Entwicklung/Cindy/SQLiteAbbildung/HUSST_Cindy2CreateSql.xsl

```diff
@@ -121,15 +121,22 @@
 		<xsl:variable name="type"><xsl:apply-templates mode="fieldtypename" select="."/></xsl:variable>
 
 		<xsl:choose>
-			<xsl:when test="$type='unsignedInt'">largeint</xsl:when>
-			<xsl:when test="$type='integer'">largeint</xsl:when>
+			<!-- SQLite kennt eigentlich nur *einen* Integer Datentyp, 64bit
+				 Manche DBKomponenten interpretieren den aber als 32bit Integer,
+				 wohingegen der 'bigint' als 64bit Integer verstanden wird.
+				 Der zuvor verwendete 'largeint' wird zumindest von der ZEOS Library, 
+				 zumindest inzwischen, nicht korrekt als 64bit sondern als 32bit Integer
+				 interpretiert.   
+			-->
+			<xsl:when test="$type='unsignedInt'">bigint</xsl:when>
+			<xsl:when test="$type='integer'">bigint</xsl:when>
 			<xsl:when test="$type='decimal'">real</xsl:when>
 			<xsl:when test="$type='float'">real</xsl:when>
 			<xsl:when test="$type='date'">datetime</xsl:when>
 			<xsl:when test="$type='dateTime'">datetime</xsl:when>
 			<xsl:when test="$type='boolean'">bit</xsl:when>
 			<xsl:when test="($type='string') or ($type='language')">
-				<xsl:if test="$maxLength">Varchar(<xsl:value-of select="$maxLength"></xsl:value-of>)</xsl:if>
+				<xsl:if test="$maxLength">varchar(<xsl:value-of select="$maxLength"></xsl:value-of>)</xsl:if>
 				<xsl:if test="not($maxLength)">string</xsl:if>
```

Comment:
string müsste bei SQLite in einen Numerischen Typ umgewanselt werden: https://www.sqlite.org/datatype3.html#determination_of_column_affinity
sollte noch auf Text geändert werden 🤔